CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - `event.config.collection` is no longer a parameter to the `DiscoverPdrs`
     task
 
+- **CUMULUS-1997**
+  - `@cmr-client/CMRSearchConceptQueue` parameters have been changed to take a `cmrSettings` object containing clientId, provider, and auth information. This can be generated using `@cmrjs/cmr-utils/getCmrSettings`.
```

Comment:
```suggestion
  - `@cumulus/cmr-client/CMRSearchConceptQueue` parameters have been changed to take a `cmrSettings` object containing clientId, provider, and auth information. This can be generated using `@cumulus/cmrjs/cmr-utils/getCmrSettings`.
```